src/modules/acl/types.ts

```diff
@@ -6,6 +6,7 @@ export type InfoResponse = {
   timestamp: Date
   expiration: number
   payload: string // This is the data to sign
+  method: 'put' | 'delete'
```

Comment:
maybe you can change this field for `action: 'grant' | 'revoke'` to be more product-related